### PR TITLE
pytest conversion - cli/test_oscap_tailoringfiles.py

### PR DESCRIPTION
**Pull request by jameerpathan111**
Test result:

```
(env-robottelo) [jpathan@jpathan robottelo]$ pytest -n 2 tests/foreman/cli/test_oscap_tailoringfiles.py| tee scap.log 
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-4.6.3, py-1.8.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jpathan/projects/robottelo
plugins: services-1.3.1, xdist-1.30.0, mock-1.10.4, cov-2.7.1, forked-1.1.3
gw0 I / gw1 I
gw0 [12] / gw1 [12]

.....s.sss.s                                                             [100%]

=============== 7 passed, 5 skipped, 8 warnings in 66.12 seconds ===============

```

### CODE REVIEW

File: tests/foreman/cli/test_oscap_tailoringfiles.py [hunk 1]
```diff
@@ -26,28 +26,29 @@
...
-    @classmethod
-    def setUpClass(cls):
-        super(TailoringFilesTestCase, cls).setUpClass()
-        cls.tailoring_file_path = file_downloader(
+    @pytest.fixture(scope="class")
+    def tailoring_file_path(self):
+        tailoring_file_path = file_downloader(
             file_url=settings.oscap.tailoring_path, hostname=settings.server.hostname
         )[0]
+        return tailoring_file_path
```
Comment (JacobCallahan) [inline tests/foreman/cli/test_oscap_tailoringfiles.py:47]: is this fixture something that could be useful outside of this file?

Comment (jameerpathan111) [reply tests/foreman/cli/test_oscap_tailoringfiles.py]: yes, thanks for noticing it. I have added aaa815b813f2bcc0a9efc79f7b89badf86ce204b commit for it.

**Comment by jameerpathan111** [comment]
Test Results:
```
$ pytest tests/foreman/api/test_oscap_tailoringfiles.py
=================================================================================== test session starts ===================================================================================
collecting ... 2020-08-27 05:00:40 - conftest - DEBUG - Collected 1 test cases
collected 1 item                                                                                                                                                                          

tests/foreman/api/test_oscap_tailoringfiles.py .                                                                                                                                    [100%]

========================================================================== 1 passed, 2 warnings in 35.59 seconds ==========================================================================
```
```
$ pytest tests/foreman/api/test_oscappolicy.py
=================================================================================== test session starts ===================================================================================
collecting ... 2020-08-27 05:13:27 - conftest - DEBUG - Collected 1 test cases
collected 1 item                                                                                                                                                                          

tests/foreman/api/test_oscappolicy.py .                                                                                                                                             [100%]

========================================================================= 1 passed, 2 warnings in 127.98 seconds ==========================================================================
```
```
$ pytest tests/foreman/cli/test_oscap_tailoringfiles.py
=================================================================================== test session starts ===================================================================================
collecting ... 2020-08-27 05:01:13 - conftest - DEBUG - Collected 12 test cases
collected 12 items                                                                                                                                                                        

tests/foreman/cli/test_oscap_tailoringfiles.py ......ss.sss                                                                                                                         [100%]

==================================================================== 7 passed, 5 skipped, 4 warnings in 102.77 seconds ====================================================================
```
```
$ pytest tests/foreman/ui/test_oscappolicy.py
=================================================================================== test session starts ===================================================================================
collecting ... 2020-08-27 05:01:27 - conftest - DEBUG - Collected 2 test cases
collected 2 items                                                                                                                                                                         

tests/foreman/ui/test_oscappolicy.py ..                                                                                                                                             [100%]

========================================================================= 2 passed, 6 warnings in 649.14 seconds ==========================================================================

```
```
$ pytest tests/foreman/ui/test_oscaptailoringfile.py
=================================================================================== test session starts ===================================================================================
collecting ... 2020-08-27 05:01:45 - conftest - DEBUG - Collected 4 test cases
collected 4 items                                                                                                                                                                         

tests/foreman/ui/test_oscaptailoringfile.py .sss                                                                                                                                    [100%]

==================================================================== 1 passed, 3 skipped, 4 warnings in 146.40 seconds ====================================================================

```

File: tests/foreman/cli/test_oscap_tailoringfiles.py (hunk 1 above)
Comment (JacobCallahan) [reply tests/foreman/cli/test_oscap_tailoringfiles.py]: Thank you for externalizing it, Jameer!

**Review by mirekdlugosz — APPROVED** [review]
ACK

**Pull request merged**